src/components/Sketches/components/ImageSelector.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import ReactModal from "react-modal";
+import { Container } from "reactstrap";
+class ImageSelector extends React.Component {
+  render(){
+    return(
+      <ReactModal
+        isOpen={this.props.isOpen}
+        onRequestClose={this.props.closeModal}
+        className="sketches-image-modal"
+        overlayClassName="profile-image-overlay"
```

Comment:
Ideally classes like these are bundled into their own separate SCSS file (and probably have different names so we don't use `profile-image-overlay`) - it should stop us from making future mistakes when we edit CSS and don't see the side effects realized.

Do you mind making a new `ImageSelector` SCSS file and dropping all the stuff in there?